packages/frontend/src/pages/Tile/components/TableBanner/EditMode.tsx

```diff
@@ -36,14 +36,14 @@ const EditMode = () => {
     [mode],
   )
 
-  if (mode === 'edit') {
+  if (hasEditPermission) {
```

Comment:
actly why you dont want to show the edit mode badge ah

Reply:
designer choice